### PR TITLE
Do not reinstall plugins if they are already available

### PR DESCRIPTION
**Pull request by ktsakalozos**
Fixes https://github.com/juju-solutions/layer-cwr/issues/35

### CODE REVIEW

File: reactive/cwr.py [hunk 1]
```diff
@@ -72,12 +72,14 @@ def install_juju():
...
         3, True, 'https://github.com/juju-solutions/matrix')
-    # juju plugins (for crashdump)
-    utils.run_as('root', 'git', 'clone', 'https://github.com/juju/plugins',
-                 '/usr/local/lib/juju-plugins')
-    for plugin in Path('/usr/local/lib/juju-plugins').iterdir():
-        if plugin.name.startswith('juju-'):
-            (Path('/usr/local/bin') / plugin.name).symlink_to(plugin)
+    # juju plugins (for crashdump). Skip if already deployed.
+    plugins_dir = '/usr/local/lib/juju-plugins'
+    if not os.path.isdir(plugins_dir):
```
Comment (kwmonroe) [inline reactive/cwr.py]: Be nice if we did a `git pull` if the `plugins_dir` path already exists.  That would ensure the user got the latest from the plugins repo.

If we did this, you'd probably want to move the `for plugin in ... if startswith(juju) ... symlink ...` logic out of the conditional and include an `if not Path(plugin).exists()` before creating a new symlink in `/usr/local/bin`.

**Comment by petevg** [comment]
Agree w/ @kwmonroe -- we don't want to block updates when they are wanted.

File: reactive/cwr.py (hunk 1 above)
Comment (ktsakalozos) [inline reactive/cwr.py]: Done!

**Comment by kwmonroe** [comment]
LGTM, thanks!

**Pull request merged**